### PR TITLE
EN-997

### PR DESCRIPTION
**Pull request by vladimir-grezin**
Pull Request Checklist
======================

Work relates to:
----------------
Link to Jira Task/User Story/Bug/Epic:
[https://incountry.atlassian.net/browse/EN-997]

Code Quality
------------
[x] Ensure code compiles or passes linter locally.
[x] Ensure all unit tests pass.
[x] Do your best to ensure integration tests pass. This is not always possible and should not gate the merge.
[x] Remove, ignore, or mark pending any brittle tests.

### CODE REVIEW

**Comment by vladimir-grezin** [comment]
> We need to find out why TravisCI is failing before we merge the code in.

Done. There was a problem with JUnit

File: src/test/java/com.incountry/CryptoTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public class CryptoTest {
+    @Test
+    public void testEncryption() throws GeneralSecurityException, IOException {
+        Crypto crypto = new Crypto("supersecret");
+        String plainText = "some plain text";
+        String encrypted = crypto.encrypt(plainText);
+        String decrypted = crypto.decrypt(encrypted);
+        assertEquals(decrypted, plainText);
+        assertNotEquals(encrypted, plainText);
```
[the remark this answers is not in the document]
Comment (vladimir-grezin) [reply src/test/java/com.incountry/CryptoTest.java]: @samutaa Would you mind to give me a link to the related topic? I didn't manage to find the predefined data you mentioned. Thanks

File: README.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
 This is the java SDK for the InCountry storage network. Sign up for a free account at
 https://incountry.com, then note down your Environment ID and API key.
 
+Important notes
+---------------
+We've changed the encryption algorithm since version `0.3.0` so it is not compatible with earlier versions.
+
+
```
Comment (RommeroInCountry) [inline README.md]: We can remove this note as we've added backwards compatibility.

**Comment by avdyakovs** [comment]
@joelbruce-ic @samutaa could you please take a look? I can't merge without your approval and we need this one to continue.

File: src/main/java/com/incountry/Utils.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package com.incountry;
```
Comment (joelbruce-ic) [inline src/main/java/com/incountry/Utils.java]: I do not see anything inherently wrong at first glance with the code. Just out of curiosity, were there no Java libraries that already did this?

**Pull request merged**